### ISSUE TITLE
Loss on test set

### ISSUE THREAD

**Issue by salmarachidi** [havakv/pycox#187]
Hello,
I have trained a DeepHitSingle model and get the loss for the train set and the validation set when I fit the model, but I would like to know the loss for an independent test set. I have tried model.loss and model.compute_metrics but cannot find what parameters to use.

**Comment by salmarachidi** [comment]
I found a way with model.loss using pair_rank_mat for rank_mat and I get the same result for the validation loss but not for the train loss, got the same thing for multiple models. Is the train loss during fitting calculated a different way?
